### DISCUSSION TITLE
Ability to remove specific entries from your test history

### DISCUSSION THREAD

**Discussion by Kelvets** [monkeytypegame/monkeytype#2345] [Ideas] [answered]
My usual WPM is 80-90, then I did a new test with the difficult words of the previous test and my WPM tanked. To my horror I found out that this test was recorded, caused a massive dip in my WPM-over-time graph, probably lowered my average WPM reading, **and there is no seemingly no way to remove this anomaly from my data**. The site could really use the ability to remove specific outliers from your history.

**Comment by Miodec** [comment, marked as answer]
Already suggested, dont want people to hand pick their data and make their profiles perfect. 

If you want to filter out tests using more difficult word lists, open the filters and disable that language.

**Reply by Kelvets** [reply]
"If you want to filter out tests using more difficult word lists, open the filters and disable that language."

What do you mean? Temporarily change the language to something other than English before attempting a difficult test so the results get saved separately? If so, if I do a test consisting of difficult words from my previous test, it will still be the same language as the test I just did, and changing the language will give me a test with entirely different words. It doesn't work.

Also, if someone wants to make their profile perfect, they can just keep restarting the test whenever they make a mistake and before typing the test's last letter, until they get a perfect run. Not allowing them to delete bad results doesn't stop them from cheating.

**Reply by Miodec** [reply]
If you go to the account page, just above the graph there is an 'advanced' (filters) button. If you open that you can select and deselect various properties to show on the graph. Alternatively you can just click 'current settings' to set the filters to your currently active settings and tags.

**Reply by tajhans** [reply]
@Kelvets other than keeping a profile perfect what would this feature add?

**Reply by Kelvets** [reply]
@teddinotteddy your average WPM and other average data not being skewed by an outlier that was generated under much different conditions than usual.
